src/main/resources/google_checks.xml

```diff
@@ -45,7 +45,8 @@
   <module name="LineLength">
     <property name="fileExtensions" value="java"/>
     <property name="max" value="100"/>
-    <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
+    <property name="ignorePattern"
+             value="^package.*|^import.*|href=&quot;(.*?)&quot;&gt;(.*?)|http://|https://|ftp://"/>
```

Comment:
> `href=&quot;(.*?)&quot;&gt;(.*?)` how did you come up with this pattern?

tbh I'm not so good with regexs so I asked copilot to write a regex to validate href links and modified it little to match our requirements.

> Can we use something like: `href\s*=\s*\"[^\"]*\"`?

`\s*` allows us to insert whitespaces right? I guess we should not allow for whitespaces between `href` and `=` so it will make link harder to read and mostly this type of link will be used with `<a>` tag and it normally does not have whitespaces between `href` and `=`. The other part of regex looks fine.

@romani what do you think about this?

Reply:
>I guess we should not allow for whitespaces between href and = so it will make link harder to read

This is subjective, I agree with you, but still it matter of style, bunch of people will disagree with us. Better to avoid personal opinions in this area.

Reply:
okay I have updated the regex:

https://github.com/checkstyle/checkstyle/blob/e680f3439abeff21ac164d32550d8682cacc9ef2/src/main/resources/google_checks.xml#L48-L49

Updated tests:

https://github.com/checkstyle/checkstyle/blob/e680f3439abeff21ac164d32550d8682cacc9ef2/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputLineLength.java#L185-L205